### ISSUE TITLE
Should opening a definition focus it in the namespace tree?

### ISSUE THREAD

**Issue by ceedubs** [unisonweb/ui-core#45]
A common occurrence for me is that I want a method for say `Map` that I assume already exists but I don't know what it's called. I search for `Map` and see ~10 results but none of them are the function that I'm looking for. It _feels_ like I should be able to accept the top `Map` data type result and then scroll down in the left pane to look at the rest of the methods under the `Map` namespace. But the left pane isn't actually focused on `Map`, so I scroll down to `M`, realize that it's not there, scroll to `data`, click it, and then scroll to `Map`.

**Comment by hojberg** [comment]
What would "focus" mean in this suggestion? That `data` is open and `Map` has a visual indicator? Or do you mean that we should open `data` and scroll to it?

**Comment by ceedubs** [comment]
@hojberg yeah, I was talking about scrolling to it. In general as I switch between definitions I find that I expect the left side to scroll to the one that I'm on (though I still want to be able to scroll it manually; I just want it to line up when I shift focus in the right pane). But I don't know the rules of UI - maybe this would be bad.
